src/test/java/com/michelin/kafkactl/command/ApplyTest.java

```diff
@@ -413,7 +417,7 @@ void shouldNotApplySchemaWhenNotExist() {
         when(resourceService.parseResources(any(), anyBoolean(), any()))
                 .thenReturn(Collections.singletonList(resource));
         when(kafkactlProperties.getCurrentNamespace()).thenReturn("namespace");
-        doCallRealMethod().when(resourceService).enrichSchemaContent(any(), any());
+        doCallRealMethod().when(resourceService).prepareResources(any(), any());
 
         CommandLine cmd = new CommandLine(apply);
         StringWriter sw = new StringWriter();
```

Comment:
It would be nice to add a unit test that verify resources have been sorted properly. We can basically create a resource file in `src/test/resources` with a namespace, role bindings, ACLs and a topic in any order, and verify in the CLI output it has been applied in order